sdcm/utils/sstable/sstable_utils.py

```diff
@@ -150,23 +150,32 @@ def count_sstable_tombstones(self, sstable: str) -> int:
         self.log.debug('Got %s tombstones for sstable: %s', num_tombstones, sstable)
         return num_tombstones
 
+    def set_db_node_if_unset_or_down(self):
```

Comment:
What does this mean? What does it mean to "set node", could you add docstring with a bit more explanation?